cmake/Modules/FindXCCL.cmake

```diff
@@ -0,0 +1,65 @@
+# This will define the following variables:
+# XCCL_FOUND               : True if the system has the XCCL library.
+# XCCL_INCLUDE_DIR         : Include directories needed to use XCCL.
+# XCCL_LIBRARY_DIR         ：The path to the XCCL library.
+# XCCL_LIBRARY             : XCCL library fullname.
+
+include(${CMAKE_ROOT}/Modules/FindPackageHandleStandardArgs.cmake)
+
+set(XCCL_ROOT $ENV{CCL_ROOT})
```

Comment:
How do you get `CCL_ROOT`? I think you cannot assume it will be set after oneccl source.

Reply:
It will auto set after source oneccl env, and I remember oneccl update not affect this flag.